### PR TITLE
Core api

### PR DESCRIPTION
**Pull request by masone**
For the documentation focus project. Really simple start with documenting the core apis

### CODE REVIEW

File: public/guides/customization/editor/properties-panel.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Properties panel
+
+## Register angular directive
+
+```
+angular.module('livingdocs-editor').directive 'myProperties',
```
Comment (marcbachmann) [inline public/guides/customization/editor/properties-panel.md]: Thanks for improving the docs.
JS examples are a bit easier to read 😉

**Comment by peyerluk** [comment]
@masone I like these additions as well. Could you rebase to the current folder structure? Then I could merge it in.

**Comment by masone** [comment]
Rebased

**Comment by peyerluk** [comment]
I moved the content into the new folder structure and converted the code examples to javascript.

**Pull request merged**